tests/pds/dataservice_test.go

```diff
@@ -298,6 +298,47 @@ var _ = Describe("{EnableandDisableNamespace}", func() {
 
 })
 
+var _ = Describe("{RebootActiveNodeDuringDeployment}", func() {
+	JustBeforeEach(func() {
+		StartTorpedoTest("RebootActiveNodeDuringDeployment", "Reboots a Node onto which a pod is coming up", pdsLabels, 0)
+	})
+
+	It("deploy Dataservices", func() {
+		Step("Deploy Data Services", func() {
+			for _, ds := range params.DataServiceToTest {
+				if ds.Name != postgresql {
+					continue
+				}
+				Step("Start deployment, Reboot a node on which deployment is coming up and validate data service", func() {
+					isDeploymentsDeleted = false
+					// Global Resiliency TC marker
+					pdslib.MarkResiliencyTC(true)
+					// Type of failure that this TC needs to cover
+					failure := pdslib.ResiliencyFailure{
+						Type: active_node_reboot_during_deployment,
+						Method: func() error {
+							return pdslib.RebootActiveNodeDuringDeployment(params.InfraToTest.Namespace)
+						},
+					}
+					pdslib.DefineFailureType(failure)
```

Comment:
I can see `DefineFailureType ` is updating the var `FailureType ` and `FailureType ` is not consumed by any other places. Is this expected ?

Reply:
Failuretype was the struct that was used to call it's failure method. I have updated the names now as we discussed